src/PDO/SchemaPDOOracle.php

```diff
@@ -427,7 +439,7 @@ public function getLastInsertID(string $sequenceName = ''): string
             /* get the last insert id from the master connection */
             $sequenceName = $this->quoteSimpleTableName($sequenceName);
 
-            return $this->db->useMaster(function (ConnectionPDOInterface $db) use ($sequenceName) {
+            return (string) $this->db->useMaster(static function (ConnectionPDOInterface $db) use ($sequenceName) {
```

Comment:
static function(ConnectionPDOInterface $db, $sequenceName)  ?

Reply:
I thought of that myself, but the callback has only one argument: https://github.com/yiisoft/db/blob/976e42595ebfa08a7064d64cdbd957712f0b5269/src/Connection/Connection.php#L209